podaac/subsetter/subset.py

```diff
@@ -1221,6 +1225,18 @@ def subset(file_to_subset, bbox, output_file, variables=None,
 
     nc_dataset, rename_vars = dc.remove_duplicate_dims(nc_dataset)
 
+    orig_decode_cf_datetime = xarray.coding.times.decode_cf_datetime
```

Comment:
Can you put this in it's own function with a big warning comment how this is absolutely terrible and should be removed at the first possible opportunity with a link to the github discussion about it.